packages/blade/src/components/BaseMenu/BaseMenuItem/tokens.ts

```diff
@@ -0,0 +1,38 @@
+import type { Theme } from '~components/BladeProvider';
+import { size } from '~tokens/global';
+
+const itemFirstRowHeight = size[20];
+
+const getItemPadding = (
+  theme: Theme,
+): {
+  itemPaddingMobile: 4;
+  itemPaddingDesktop: 8;
+} => {
+  return {
+    itemPaddingMobile: theme.spacing[2],
+    itemPaddingDesktop: theme.spacing[3],
+  };
+};
+
+const getItemMargin = (theme: Theme): number => {
+  return theme.spacing[1];
+};
+
+const getItemHeight = (
+  theme: Theme,
+): {
+  itemHeightMobile: number;
+  itemHeightDesktop: number;
+} => {
+  return {
+    itemHeightMobile:
+      // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+      itemFirstRowHeight + getItemPadding(theme).itemPaddingMobile * 2 + getItemMargin(theme) * 2,
+    itemHeightDesktop:
+      // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
```

Comment:
lets disable globaly

Reply:
https://github.com/razorpay/blade/issues/2475#issue-2788792914

Moved it to this. We should also remove all the places where we have added comment to remove this rule. Will do it separately.